.circleci/config.yml

```diff
@@ -1,121 +1,218 @@
 version: 2
-
 jobs:
+  "percy-finalize":
+    docker:
+      - image: percyio/agent
+    steps:
+      - run: percy finalize --all
```

Comment:
this works together with a `PERCY_PARALLEL_TOTAL: '-1'` in test job, percy command will compare the number of containers by fetching ENV variable value from circleci.